api/api.yaml

```diff
@@ -150,4 +150,25 @@ paths:
                 items:
                   $ref: '../result/flow.yaml#/components/schemas/Result.Flow'
 
-
+  /results/bgpinfo:
+    description: >-
+      BGP results API
+    post:
+      tags: [Results]
+      operationId: get_bgp_infos
+      requestBody:
+        description: >-
+          The request to retrieve BGP Router statistics and learned routing information.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../result/bgpinfo.yaml#/components/schemas/Result.BgpInfoRequest'
+      responses:
+        '200':
+          description: >-
+            BGP Router statistics and learned routing information
+          content:
+            application/json:
+              schema:
```

Comment:
return is an array of items, see the return result for /results/port and /results/flow

Reply:
Thanks!